core/src/main/java/org/springframework/security/oauth2/server/authorization/authentication/OAuth2AccessTokenAuthenticationToken.java

```diff
@@ -25,6 +25,7 @@
 
 /**
  * @author Joe Grandja
+ * @author Madhu Bhat
  */
 public class OAuth2AccessTokenAuthenticationToken extends AbstractAuthenticationToken {
```

Comment:
Javadoc for class is required as per issue requirements.

Reply:
Added only to the getter that I added and not to the class as there is an open PR with javadoc for the class.